tests/transforms/test_decompose_transform_graph.py

```diff
@@ -13,14 +13,15 @@
 # limitations under the License.
 
 """Tests the ``decompose`` transform with the new experimental graph-based decomposition system."""
-
-# pylint: disable=no-name-in-module, too-few-public-methods
+from functools import partial
 
 import numpy as np
 import pytest
 
 import pennylane as qml
 
+# pylint: disable=no-name-in-module, too-few-public-methods
```

Comment:
Are these warnings localized to a single test/test class? If so, I'd prefer them to be put there instead of globally suppressing the warnings.